spec/models/user_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe '#name_or_email' do
+    subject { user.name_or_email }
+
+    context '名前があるとき' do
+      let(:user) { User.new(name: 'テスト 太郎', email: 'user@example.com') }
```

Comment:
```suggestion
      let(:user) { craete(:user, name: 'テスト 太郎', email: 'user@example.com') }
```

FactoryBotを使う、で一貫性を持たせた方がFBC生が混乱しにくいかも

Reply:
これは自分の対応漏れでした 🙇 FactoryBotを使う方向で統一しました。